app/views/users/show.html.haml

```diff
@@ -0,0 +1,37 @@
+%section.user.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest(@user.email.downcase)}?s=40")
+          %h2 #{@user.full_name}'s video collection (#{@user.queue_items.size})
+          = link_to 'Follow', follow_path(id: @user.id), method: :post, class: 'btn btn-default' unless (@user == current_user || current_user.followed_users.include?(@user))
+        %table.table
+          %thead
+            %tr
+              %th(width="30%") Video Title
+              %th(width="15%") Genre
+          %tbody
+            - @user.queue_items.each do |queue_item|
+              %tr
+                %td
+                  = link_to queue_item.video_title, queue_item.video
+                %td
+                  = link_to queue_item.category_name, queue_item.category
+
+%section.user_reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %header
+        %h3 #{@user.full_name}'s reviews (#{@user.reviews.size})
+      %ul
+        - @user.reviews.each do |review|
+          %article.review
+            %li.row
+              .col-sm-2
+                %p
+                  = link_to "\"#{review.video_title}\"", review.video
```

Comment:
You are allowed to do this `= link_to review.video_title, review.video` instead. You don't have to interpolate the title in.